metpy/calc/kinematics.py

```diff
@@ -475,9 +475,14 @@ def ageostrophic_wind(heights, f, dx, dy, u, v, dim_order='yx'):
     If inputs have more than two dimensions, they are assumed to have either leading dimensions
     of (x, y) or trailing dimensions of (y, x), depending on the value of ``dim_order``.
 
+    The order of the inputs will be changed in 1.0 to be (heights, u, v, f, dx, dy).
+    To updated to the new format, use `from metpy.future import ageostrophic_wind`.
+
     """
-    u_geostrophic, v_geostrophic = geostrophic_wind(heights, f, dx, dy, dim_order=dim_order)
-    return u - u_geostrophic, v - v_geostrophic
```

Comment:
Could you put the `from ..future import ageostrophic_wind as _ageostrophic_wind` here to break the circular import?